### PR TITLE
Create _pins.allsoppo.json

### PR DESCRIPTION
**Pull request by allsoppo**
add my pin to the map!

### CODE REVIEW

**Comment by Wispring** [comment]
Hi allsoppo, 
you need to change this file.`
+---
+githubHandle: allsoppo  
+latitude: 30.267153
+longitude: -97.743061
+---
Just open it again and at the end of the last +--- press enter on your keyboard. This self-completes the file and is what travis-ci is looking for and needs to process your file w/o failing a file check`

**Comment by Wispring** [comment]
> +---
+githubHandle: allsoppo  
+latitude: 30.267153
+longitude: -97.743061
+---
which is 5 lines long needs to be 6 lines long. simply opening the file. putting your cursor on the end of the second +--- and pressing enter on your keyboard will accoplish this. I had the same problem when I first did this :)

File: _pins.allsoppo.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: allsoppo  
+latitude: 30.267153
+longitude: -97.743061
+---
```
Comment (Wispring) [inline _pins.allsoppo.json]: see. this file now only has 5 lines. place your cursor at the end of this line to make it 6 lines and this is what that travis.ci code parser is looking for and why your file is failing to process through.

Comment (Wispring) [inline _pins.allsoppo.json]: I need to go to work now. If you read this and add that one line it should get through(if there are no other errors). I had the same problem a few days ago and doing this is what worked for me. Hope it works for you as well. cheers.
respectfully, wisp

Comment (allsoppo) [inline _pins.allsoppo.json]: Thank you! It worked :)

**Comment by allsoppo** [comment]
@githubteacher Why is this commit still failing?

**Comment by hollenberry** [comment]
Hey @allsoppo ! Almost there! It looks like the only thing left that needs changing is your file location. 

Instead of `_pins.allsoppo.json` it should be `_pins/allsoppo.json` 👍 

Let me know if you have any other concerns!

**Comment by hollenberry** [comment]
@allsoppo You're all set!! Go ahead and click that big merge button! There were some trailing spaces on line 2, so I yanked those out for you, but everything else was perfect. Well done!

**Pull request merged**